detectron2/projects/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
-import importlib
+from importlib import abc, util
```

Comment:
`abc` and `util` can confused with other modules, it's better to avoid using them without package name.
```suggestion
import importlib.abc
import importlib.util
# or from importlib.abc import MetaPathFinder, ...
```

Reply:
Yeah; related to this, adding a `__all__ = []` to the file would be helpful as well

Reply:
Fixed this as your suggestion. PTAL.